### ISSUE TITLE
Use meta fields in filter

### ISSUE THREAD

**Issue by joarwilk** [graphcool/feature-requests#220]
Right now the _xMeta field is not included in the filter input type. I think it would be super useful in certain situations. For example, lets say I want to get all users that have created at least one post:

```
query {
  allUsers(filter: {
    _postsMeta: {
      count_gt: 0
    }
  })
}
```

**Comment by joarwilk** [comment]
If there is any way to do this right now that doesnt include client side filtering please let me know 🙂

**Comment by marktani** [comment]
Closed in favor of #110. In the comment chain there's also a working syntax for your specific example.
